en/docs/get-started/overview.md

```diff
@@ -1,95 +1,64 @@
+
+- **To give the right access to the right people** - Having said that accessing an application should be convenient, you might not need all your users accessing all the applications. For example, you wouldn't want everyone accessing the payroll information and would want to give the access only to a dedicated finance team. This is just one example where users need to be authenticated based on their role. 
+
+- **To enforce strong authentication** - Authenticating a user using their username and password alone might not be the best idea with increasing phishing atttacks and data breaches. WSO2 Identity Server offers multi-factor and adaptive authentication capabilities on top of its default basic authentication. It comprehensively supports MFA with authenticators such as Email OTP, SMS OTP, FIDO etc. While ensuring maximum security using MFA, it also does not want to compromise on the usability, hence providing a number of factors based on which additional authentication would be required like the volatile nature of user devices, networks, locations, and usage contexts. The users who are not prone to mishaps via any of these factors can just skip the additional steps of authentication.  
+
+- **To empower users to manage their own accounts** - Your company might be working in a very dynamic industry where you cannot sit around waiting for things to happen. WSO2 Identity Server provides an option for users to register to applications by themselves, view, and manage their profiles. This reduces the waiting time of the application user while also taking some burden off the administrator who has to take care of the overwhelming task of onboarding all the users. 
+
+![iam functionality](../../../assets/img/guides/iam-functionality.png)
+
+- **To manage users and their accounts** - WSO2 Identity Server takes care of the whole identity lifecycle management for each account right from onboaring users, managing their profiles, managing passwords, locking or unlocking accounts, suspending accounts, and disabling accounts. You can also provision users from WSO2 Identity Server to a trusted identity provider such as Google or Salesforce, or provision users or groups to the WSO2 Identity Server’s userstore via an external application. 
+
+- **To secure ever-growing APIs** - Gone are the days where API security simple meant a secure authorization. Any organization that operates based on API interactions, needs to implement a secure framework around it. This is a expecations of both customers and regulatory authorities. While building an ecosystem for API security, WSO2 Identity Server plays a key role as the Authorization Server that supports several OAuth related standards or profiles. It supports open stanards such as OAuth, OpenID Connect, SAML 2.0, and XACML, and has the ability to integrate with heterogeneous userstores such as LDAP, Active Directory, and JDBC. It also enables hight availability, failover, and performance for a smooth operation. 
```

Comment:
`a expecations` ->`an expecation`

Reply:
typo `hight`

Reply:
`and has the ability to integrate with heterogeneous userstores such as LDAP, Active Directory, and JDBC.` is this something we should talk about in **To secure ever-growing APIs** ?